chat-plugins/github.js

```diff
@@ -0,0 +1,94 @@
+/*
+	Github plugin by Spandamn
+	This is based on https://github.com/Ecuacion/Pokemon-Showdown-Node-Bot/blob/master/features/github/index.js
+	Yes, some of the code is copied so credits to xfix, Ecuacion, jd, and anyone who's name I forgot to put here
+*/
+'use strict';
+
+let config = {};
+
+if (!Config.github) {
+	return;
+} else if (Config.github === {}) {
+	config.port = 3420;
+	config.secret = "";
+} else {
+	config = {
+		port: Config.github.port,
+		secret: Config.github.secret,
+	};
+}
+
+let git = exports.github = require('githubhook')(config);
+
+let updates = {};
+
+let sendReport = function (html) {
+	if (Config.github && Config.github.rooms) {
+		Object.keys(Config.github.rooms).forEach(room => {
```

Comment:
I'm pretty sure CONTRIBUTING.md says to look to using `for-of` over `for-in`. In any case, I would personally just use a regular `for` loop.